modules/olm-cs-status-cli.adoc

```diff
@@ -0,0 +1,119 @@
+
+. When a catalog source is created in a namespace, a pod for the catalog source is created in that namespace. To get more details about catalog source pods, list the pods in the namespace where your catalog source was created:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc get pods -n {global_ns}
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                    READY   STATUS             RESTARTS   AGE
+certified-operators-cv9nn               1/1     Running            0          36m
+community-operators-6v8lp               1/1     Running            0          36m
+marketplace-operator-86bfc75f9b-jkgbc   1/1     Running            0          42m
+example-catalog-bwt8z                   0/1     ImagePullBackOff   0          3m55s
+redhat-marketplace-57p8c                1/1     Running            0          36m
+redhat-operators-smxx8                  1/1     Running            0          36m
+----
++
+In this example output, the status for the `example-catalog-bwt8z` pod is `ImagePullBackOff`, which indicates that there is an issue pulling the catalog source's index image.
+
+. Use the `oc describe` command to inspect a pod for more detailed information:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc describe pod example-catalog-bwt8z -n {global_ns}
+----
++
+.Example output
+[source,terminal,subs="attributes+"]
+----
+Name:         example-catalog-bwt8z
+Namespace:    {global_ns}
+Priority:     0
+Node:         ci-ln-jyryyg2-f76d1-ggdbq-worker-b-vsxjd/10.0.128.2
+...
+Events:
+  Type     Reason          Age                From               Message
+  ----     ------          ----               ----               -------
+  Normal   Scheduled       48s                default-scheduler  Successfully assigned {global_ns}/example-catalog-bwt8z to ci-ln-jyryyf2-f76d1-fgdbq-worker-b-vsxjd
+  Normal   AddedInterface  47s                multus             Add eth0 [10.131.0.40/23] from openshift-sdn
+  Normal   BackOff         20s (x2 over 46s)  kubelet            Back-off pulling image "quay.io/example-org/example-catalog:v1"
+  Warning  Failed          20s (x2 over 46s)  kubelet            Error: ImagePullBackOff
+  Normal   Pulling         8s (x3 over 47s)   kubelet            Pulling image "quay.io/example-org/example-catalog:v1"
+  Warning  Failed          8s (x3 over 47s)   kubelet            Failed to pull image "quay.io/example-org/example-catalog:v1": rpc error: code = Unknown desc = reading manifest v1 in quay.io/example-org/example-catalog: unauthorized: access to the requested resource is not authorized
+  Warning  Failed          8s (x3 over 47s)   kubelet            Error: ErrImagePull
+----
++
+In this example output, the error messages indicate that the catalog source's index image is failing to pull successfully because of an authorization issue. For example, the index image might be stored in a registry that requires login credentials.
```

Comment:
Friendly reminder to repeat "In the preceding example output" or ignore the sugg.